### ISSUE TITLE
check everyones status every hour

### ISSUE THREAD

**Issue by c-harlotte** [c-harlotte/yaycat#7]
so i have an idea, the bot checks everyones status every hour and if it contains the text ".gg/catstare" in it the bot gives them the role "awesome". and it also checks the people who have the role awesome every 30 minutes or so and if their status doesnt contain ".gg/catstare" any more it removes the role. 

i have no idea how to do that

**Comment by c-harlotte** [comment]
help

**Comment by deathawp** [comment]
idk how d o that Lol

**Comment by f7nl4y** [comment]
learn js get good noob

**Comment by c-harlotte** [comment]
ill drop ur dox and ur door

**Comment by deveopinghitloa** [comment]
Already made a bot like this, but it will automatically work when someone adds in in their status. get good skid

**Comment by c-harlotte** [comment]
and who are you again?

**Comment by deveopinghitloa** [comment]
read my name lol

**Comment by deathawp** [comment]
> Already made a bot like this, but it will automatically work when someone adds in in their status. get good skid

u have only forked repos on ur profile :dead:

**Comment by c-harlotte** [comment]
true

**Comment by deveopinghitloa** [comment]
yes all forked and improved idek why ur talking when your having issues
making something simple you retard 🤣🤣🤣

**Comment by c-harlotte** [comment]
ok go back to "deveoping" stuff lmao who tf even are you

**Comment by deveopinghitloa** [comment]
ask sigma who lxi is and there’s your answer 🤣🤣🤣

**Comment by c-harlotte** [comment]
ur mom

**Comment by deveopinghitloa** [comment]
so no actual response to your incompetence? ok 🤨

**Comment by c-harlotte** [comment]
ur mom :troll:

also, i never tried to actually make it, if id really want to i probably can but then the code is gonna be horrible

**Comment by deveopinghitloa** [comment]
why can’t you code good? is it too hard for you? 🤣🤣🤣🤣🤣🤣🤣🤣🤣

**Comment by c-harlotte** [comment]
it actually is. i am still learning javascript and all of the different frameworks and i am not entirely focused on learning to code right now since i do other stuff

**Comment by PringlePot** [comment]
Bad developer

**Comment by c-harlotte** [comment]
true

**Comment by Pixwl** [comment]
@c-harlotte if you want, I can fork this repo and get the option working

**Comment by c-harlotte** [comment]
yes that would be really cool

**Comment by Pixwl** [comment]
haha I've only just read my discord notifications, sorry!

I'm quite busy atm but I can do it in like a week.
